src/thunks/fetchMovies.js

```diff
@@ -7,7 +8,7 @@ export const fetchMovies = (url) => {
       const movies = await response.json()
       dispatch(loadMovies(movies))
     } catch(error) {
-      throw new Error()
+      loading(true)
     }
```

Comment:
So here, we would want to dispatch loading(true) on line 7 before the fetch call. So then our global state is in loading mode. Then after we await the fetch and dispatch loadMovies, we would set loading to false (line 13 after the catch). 
Inside the catch would be where we would dispatch(isError(true).